Packs/Orca/Integrations/Orca/Orca.yml

```
@@ -3,8 +3,8 @@ commonfields:
   id: Orca
   version: -1
 configuration:
-- display: API Key
-  name: apikey
+- display: API Token
+  name: api_token
   required: true
   type: 4
```

per discussion with @edik24 , change to `type: 9`
```suggestion
  type: 9
```